scantool/diag_l0_elm.c

```diff
@@ -57,6 +57,9 @@ struct elm_device {
 
 	struct diag_serial_settings serial;
 	ttyp *tty_int;			/** handle for tty stuff */
+
+	uint8_t kb1, kb2;	// key bytes from 5 baud init
+	uint8_t atsh[3];	// current header setting for ISO9141
```

Comment:
this should be initialized explicitly somewhere even if it will probably work fine. The code that checks if headers were changed is otherwise reading an uninitialized value on first execution